wagtail_airtable/mixins.py

```diff
@@ -189,10 +190,17 @@ def match_record(self) -> str:
             airtable_column_name = self.AIRTABLE_UNIQUE_IDENTIFIER
 
         records = self.client.search(airtable_column_name, value)
-        if len(records) == 1:
-            # Only one record to return.
-            # TODO Handle more than one result from the airtable.seach() method
+        total_records = len(records)
+        if total_records:
+            # If more than 1 record was returned log a warning.
+            if total_records > 1:
+                logger.warning(
```

Comment:
Im not quite sure what you mean here 🤔 This will only log the warning but still use the first available record it can find and then map that record_id to the model so future saves and imports are kept in sync with that one particular row. 

But if there are say, 5 rows, all with the same slug (as an example), the one it chooses from the Airtable API would be a random one — I had noticed there isnt consistency in the results that come back from Airtable. But the one that's chosen is the one that's used from them forward because it'll look for the record_id vs. a column lookup. 

I think we should keep this open for discussion!

Reply:
Sorry, my phrasing was confused here. What I mean is: maybe we should only log a warning and not import at all here. But what you say with `record_id` makes sense - I'd forgotten this was only done initially, and mapping from then on uses the id.